### ISSUE TITLE
Segment Play on Scroll Action

### ISSUE THREAD

**Issue by AntonioXIII** [LottieFiles/lottie-interactivity#55]
## Overview & Motivation

I wanted to play only a portion of an animation when the animation became visible to a user. The following code, based off your examples, allowed me to play the entire animation:

```
var lott = LottieInteractivity.create({
      player:'#targetLottie',
      mode:"scroll",
      actions: [
          {
            visibility: [0.0, 1.0],
            type: "play"
          }
      ]
    });
```
**However**, I wanted to play only a portion of the animation. It would be nice if, like in other modes, I could specify a `frames` option or add a `segmentPlay` type so that I could specify a segment of frames I wanted played. I was able to accomplish this on my own, but it would be nice to have this feature in the library. 

Thank you for your consideration.

## Type

- [ ] New feature
- [x] Changes to existing features

## Labels

- [ ] Add the `Type: Feature Request` label to this issue.

**Comment by samuelOsborne** [comment]
Hi @AntonioXIII, thanks for making this issue, it should be fixed in the latest version v1.3.4, cheers!

**Comment by AntonioXIII** [comment]
@samuelOsborne Wow, incredible turn-around time! Thank you very much!
